src/components/BookList.js

```diff
@@ -12,9 +13,14 @@ const AvailablilityIcon = ({checkedOutBy}) => {
 }
 
 const BookLink = (props) => {
-    const { bookId, title, checkedOutBy } = props.book;
+    const { bookId, title, checkedOutBy, isbn } = props.book;
+    const bookData = useBookData(isbn);
+
+    const imageUrl = (bookData && bookData.cover) ? bookData.cover.small : '';
+
     return (
         <li className={styles.listItem}>
+            <img src={imageUrl} alt={`Cover for ${title}`} data-testid={`${isbn}-cover`} />
```

Comment:
Should we just not display the image if there isn't one (instead of an empty url)?

Reply:
I like this. I'll change the code to reflect this